navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationOptions.java

```diff
@@ -76,6 +80,7 @@ public static Builder builder() {
       .snapToRoute(true)
       .manuallyEndNavigationUponCompletion(false)
       .defaultMilestonesEnabled(true)
+      .metersRemainingTillArrival(NavigationConstants.METERS_REMAINING_TILL_ARRIVAL)
```

Comment:
Do we want to open this up as a new navigation option as opposed to using the maneuver zone radius for now?

Reply:
I think it makes sense to have this as a separate option and has little to no cons to leaving it like this.